cmd/service.go

```diff
@@ -102,10 +104,10 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Received signal to exit.")
+			console.Println("Received signal to exit.")
 			go func() {
 				time.Sleep(serverShutdownPoll + time.Millisecond*100)
-				log.Println("Waiting for active connections to terminate - press Ctrl+C to quit immediately.")
+				console.Println("Waiting for active connections to terminate - press Ctrl+C to quit immediately.")
```

Comment:
What do you mean by `formatting of messages`?  Have you seen `log.Println()` doing extra formatting?